src/pycram/datastructures/pose.py

```diff
@@ -285,25 +331,6 @@ def multiply_quaternions(self, quaternion: List) -> None:
 
         self.orientation = (x, y, z, w)
 
-    def set_orientation_from_euler(self, axis: List, euler_angles: List[float]) -> None:
```

Comment:
because it exists in tf.transformations, see [here](https://github.com/cram2/pycram/pull/193/files/c3d876171d27a010cf9dc044473d30cf26cd39cb#diff-dd20dbcb75597933e9c10777339c7c7b3e49200eb2bc92bc1321a20097716a8f:~:text=Union%2C%20Optional-,from%20tf.transformations%20import%20euler_from_quaternion,-from%20typing_extensions%20import).